prime-router/settings/organizations.yml

```diff
@@ -2504,6 +2504,26 @@
         port: 22
         filePath: ./upload
         credentialName: DEFAULT-SFTP
+    - name: UNIVERSAL_PIPELINE
+      externalName: Ignore FHIR Universal Pipeline
+      organizationName: ignore
+      topic: full-elr
+      customerStatus: active
+      jurisdictionalFilter: [ "%performerState = 'CO' or %patientState = 'CO'" ]
```

Comment:
Historically, this has been always set to something like `matches(ordering_facility_county, HL7_BATCH)` for receivers used in the smoke tests. Could we do that here too?